src/reanimated2/Sensor.ts

```diff
@@ -49,6 +49,8 @@ export default class Sensor<T> {
   register(
     eventHandler: ShareableRef<T> | ((data: Value3D | ValueRotation) => void)
   ) {
+    // prevent another sensor registration at the same time
+    this.sensorId = -2;
```

Comment:
This looks like a magic number, i think that the default value of sensorId should be null or undefined